server/backend/opensearch_security_client.ts

```diff
@@ -69,6 +72,7 @@ export class SecurityClient {
       const esResponse = await this.esClient
         .asScoped(request)
         .callAsCurrentUser('opensearch_security.authinfo', {
+          [AUTH_TYPE_PARAM]: authRequestType,
```

Comment:
Just curious - is there a reason you put this in a list? Or is that syntax for something else? Why can't it just be AUTH_TYPE_PARAM?

Reply:
this is not a list, it evaluate the constant: `AUTH_TYPE_PARAM ` to be `auth_type`. Without this it would literally set `AUTH_TYPE_PARAM` as the key.